src/SmartComponents/DriftPage/__tests__/__snapshots__/DriftPage.tests.js.snap

```diff
@@ -7345,14 +7367,15 @@ exports[`ConnectedDriftPage should render with error alert 1`] = `
                                                             </DropdownItem>,
                                                           ]
                                                         }
-                                                        isFlipEnabled={false}
+                                                        isFlipEnabled={true}
```

Comment:
Is it really supposed to change here?

Reply:
Patternfly had to be updated in order to update to frontend-components-utilities v 3.5.0. This is probably where all of this is coming from.